crates/agent/src/connector_tags.rs

```diff
@@ -143,11 +132,8 @@ impl TagHandler {
         .await?;
 
         if let Some(oauth2_spec) = oauth2_spec {
-            agent_sql::connector_tags::update_oauth2_spec(
-                row.connector_id,
-                oauth2_spec,
-                txn,
-            ).await?;
+            agent_sql::connector_tags::update_oauth2_spec(row.connector_id, oauth2_spec, txn)
```

Comment:
Are you running rustfmt with default settings? Seeing some churn here and elsewhere that I wouldn't expect.

Reply:
I haven't configured rustfmt, just using the built in vscode plugin... I do see this churn. I'll manually reset these couple of lines, but I _have_ seen this before too especially around autogenerated rust protobuf files...